### PR TITLE
Contrast error in issue #439

### PR DESCRIPTION
**Pull request by Chimatara**
I reverted the changes I made in the bulma/css file. I was able to fix the contrast error in issue #439 by making the changes in the custom.scss file.

### CODE REVIEW

File: _data/ols-4-schedule.yaml [hunk 1]
```diff
@@ -221,7 +221,6 @@ weeks:
         - Create a GitHub repository
```
Comment (bebatut) [inline _data/ols-4-schedule.yaml:221]: Can you revert your changes on this file?

File: css/custom.scss [hunk 2]
```diff
@@ -330,7 +330,10 @@ $link-active: $link-hover;
 .peoples {
   margin-top: 1em;
 }
-
+.code {
```
Comment (bebatut) [inline css/custom.scss:333]: Wave still complains about the contrast error: 
image

Comment (Chimatara) [reply css/custom.scss]: Hi!
I just made a commit and reverted the changes.

File: css/custom.scss [hunk 3]
```diff
@@ -331,6 +331,11 @@ $link-active: $link-hover;
   margin-top: 1em;
 }
 
+code {
```
Comment (NPDebs) [inline css/custom.scss:334]:
```suggestion
code {
  background-color: #DBDBDB;
  color:  #85001B;
  font-weight: bold;
}
```

**Review by NPDebs — CHANGES_REQUESTED** [review]
Thanks for the work here!

@bebatut
It looks good...and Wave agrees too. :)
contrast-error-pr

However, we can maintain something similar to the current colour, but with a better contrast.

```
code {
  background-color: #DBDBDB;
  color:  #85001B;
  font-weight: bold;
}
```
contrast-pr

File: css/custom.scss [hunk 4]
```diff
@@ -331,6 +331,15 @@ $link-active: $link-hover;
...
+  background-color: #DBDBDB;
+  color:  #85001B;
+  font-weight: bold;
+}
+  background-color: #1b1c1e;
+  color: whitesmoke;
+}
```
Comment (bebatut) [inline css/custom.scss:341]:
```suggestion
```

**Comment by bebatut** [comment]
Thanks a lot!

**Comment by NPDebs** [comment]
My pleasure. 😊

**Pull request merged**